nusslein_vsrp/latex/overview.tex

```diff
@@ -466,7 +412,74 @@ \subsection{Example II}\label{sec:example_lin}
 $\alpha$ can be narrowed down.
-An objective function is also needed to make the choice unique. This should be designed in a way to prefer weights that are close to the original weights.
+An objective function is also needed to make the choice unique. This should be
+designed in a way to prefer weights that are close to the original weights.
+
+
+We see that the choice of $\bt$ is subject to linear equality and inequality
+constraints.  If we choose a linear objective function, the resulting problem
+for finding the modified weights is a linear program, which can be efficiently
+solved by standard algorithms.  A natural choice of objective function is
+$$
+\text{minimize } \|\bt - b\|_1.
+$$
+The resulting problem can be phrased as an LP by using slack variables.
+In general, this LP may not have a solution; we can relax the constraints
+by requiring a lower order of consistency than the design order of the
+method.  These choices and alternatives will be considered in Section~\ref{sec:LP}.
+
+In contrast to other projection methods
+\cite{shampine1986conservation,sandu2001positive},
+minimizing the deviation of the weights instead of the deviation
+of the projected solution is computationally much more efficient
+for large systems, arising for example in the discretization of
+PDEs.
+
+
+\subsection{Example II}\label{sec:example_reac}
+
+To illustrate the usage of the method we consider the reaction system
+\cite{kopecz_comparison_2019}
+\begin{subequations}
+\label{eq:Reaction}
+\begin{align}
+u_1' &= 0.01u_2 + 0.01 u_3 +0.003u_4 - \frac{u_1 u_2}{0.01+u_1}, \\
+u_2' &= \frac{u_1u_2}{0.01+u_1}-0.01 u_2-0.5(1-\exp(-1.21 u_2^2)) u_3 -0.05 u_2, \\
+u_3' &= 0.5(1-\exp(-1.21u_2^2)) u_3 - 0.01 u_3 -0.02 u_3, \\
+u_4' &=0.05 u_2 + 0.02 u_3 + 0.003u_4,
+\end{align}
+\end{subequations}
+with initial conditions
+\begin{equation}
+u(0) = (8,2,1,4)^T.
+\end{equation}
+When solved with the Cash-Karp (CK5) \cite{cash1990variable} method and $\dt = 0.005$ the approximated solution contains negative values. This causes qualitatively wrong solutions to the problem.
+In Figure\,\ref{fig:exampleI} the obtained results are plotted with dashed lines.
+At $t=1.905$ the value of $u_1$ gets negative. This leads to a diverging solution.
+
+Now the weights are adapted. The adapted weights are of 4th order. The results are also plotted in Figure\,\ref{fig:exampleI}, with solid lines.
+The positivity constraint is now fulfilled. A qualitatively correct solution is obtained.
+
+\begin{figure}[ht]
+    \centering
+    \includegraphics[width=0.75\textwidth]{plots/exampleI.pdf}
+    \caption{Numerical approximation of the reaction problem \eqref{eq:Reaction} computed with CK5 and $\dt = 0.005$. The dashed lines are the approximations obtained without the adaption of the weights. The lower plot shows the adaptation of the weights. }
+    \label{fig:exampleI}
+\end{figure}
+
+The difference $\|\bt-b\|$ is also plotted in Figure\,\ref{fig:exampleI}.
```

Comment:
We might add the subscript `1` in `$\|\bt-b\|_1$`.

Reply:
Fixed.